src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/monitor/ProjectCreationSupervisor.scala

```diff
@@ -49,10 +49,14 @@ class ProjectCreationSupervisor(billingAccount: String, dbRef: DbReference, goog
   }
 
   def createProject(projectName: String): Unit = {
-    val newProjectMonitor = context.actorOf(ProjectCreationMonitor.props(projectName, billingAccount, dbRef, googleDAO, pollInterval), monitorName(projectName))
```

Comment:
subtle thing to note: `context.actorOf` makes a child actor of this actor. `system.actorOf` makes a new actor at the top level. unless you're deliberately designing actor hierarchies you probably want `system.actorOf`

Reply:
this also affects actor lookup: `/user/parent/newActor` with `context.actorOf`, but `/user/newActor` with `system.actorOf`